### ISSUE TITLE
How do I provoke an error?

### ISSUE THREAD

**Issue by ferhtgoldaraz** [richardscarrott/require-error-handler-webpack-plugin#4]
I'm returning a 403 Forbidden for a webpack resource, and neither the success not the fail callbacks are being called. How do I return something that will trigger the error?

**Comment by etienne-dldc** [comment]
I have the same issue, the plugin does nothing. (with webpack 1.12.12)
I use `module.ensure()` with a chunck name.

**Comment by jharris4** [comment]
If you run the example, you'll get a few output bundles files:
- output.js
- 1.output.js (generated dynamic bundle, number at start may be different)
- 2.output.js (generated dynamic bundle, number at start may be different)

If you open index.html, you should see a bunch of `success` messages in the console.

Now, if you delete any of the dynamic output bundles and reload index.html, you should see a bunch of `error` messages in the console.

So basically, this plugin allows you to do custom error handling for the case where the dynamic bundles could not be downloaded (404).

I doubt it'll work if you use 403, but still return the content...

**Comment by etienne-dldc** [comment]
Personally, I have solve this by using webpack 2. Since `require.ensure` return a Promise, I just have to catch the error.

**Comment by jharris4** [comment]
Yep, that's the easy way to do it for webpack 2. In fact, I made a pull request to this project that basically lets you pass the callback function, and then it'll automatically get added as a .catch on the promise...
